src/btree/row_srch.c

```diff
@@ -142,6 +149,62 @@ __wt_search_insert(
     cbt->compare = -cmp;
     cbt->ins = (ins != NULL) ? ins : last_ins;
     cbt->ins_head = ins_head;
+
```

Comment:
@korteland this section is the validation code you were asking about, plus the prototype at the top of the file.